deployment/distribution/build.gradle

```diff
@@ -15,7 +15,6 @@ dependencies {
     compile project(':external:api')
     compile project(':ctk:common')
     compile project(':ctk:idp')
-    compile project(':deployment:suites')
```

Comment:
✏️ There's a section for the suites directory in the README that should be removed.